resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/CircuitBreakerConfig.java

```diff
@@ -749,5 +760,13 @@ public CircuitBreakerConfig build() {
                 .orElseGet(() -> recordExceptionPredicate != null ? recordExceptionPredicate
                     : DEFAULT_RECORD_EXCEPTION_PREDICATE);
         }
+
+        private IntervalFunction validateWaitIntervalFunctionInOpenState() {
+            if (waitIntervalFunctionInOpenStateAlreadySet) {
```

Comment:
If I set config like below, doesn't it throw exception?
```java
    CircuitBreakerConfig.custom()
            .waitDurationInOpenState(Duration.ofMillis(3333))
            .build();
```

Reply:
give me time i will fix it today